keyboards/tdcleft104/info.json

```diff
@@ -0,0 +1,119 @@
+{

```

Comment:
Please use Unix line endings for all files.

Reply:
So put "+ {" at the end of all the files for the keyboard?

Reply:
No. You need to use Unix line endings, not Windows.

Reply:
It looks like you've used the GitHub web editor to create these files, which doesn't seem to respect the line ending settings of the repo. So you will need to clone it locally and fix it yourself: https://docs.qmk.fm/#/newbs

Reply:
I used sublime to edit these files and then uploaded them but just have been editing in this thread. So do I just need to fix the json or all the files?

Reply:
info.json and rules.mk. Those are the ones that I can see are using Windows line endings:
![image](https://user-images.githubusercontent.com/4781841/186067227-164c9d66-ddce-41db-919f-1d6af5ae92ad.png)
There are two lines where there should be one. It doesn't matter that you used Sublime, because the line endings don't transfer over when you copy and paste into GitHub. You have to commit the changes directly using Git.